contrib/kind.sh

```diff
@@ -34,6 +34,37 @@ delete()
   kind delete cluster --name ${KIND_CLUSTER_NAME:-ovn}
 }
 
+update_kubeconfig()
+{
+  # For IPv6 NO-HA Shared-Gateway Mode, it is a pure IPv6 network and no IPv4 connectivity.
+  # So need to updae the KubeConfig with an IPv6 address. In HA mode, there is a load-balancer
+  # in front which responds to the IPv4 requests.
+  if [ "$IP_FAMILY" == "ipv6" ] && [ "$OVN_GATEWAY_MODE" == "shared" ] && [ "$KIND_HA" == false ]; then
+    # Determine if API_URL has an IP Address or Container Name
+    SERVER=`echo "$API_URL" | sed -e "s@^https://@@" -e "s@:6443@@"`
+    if [[ "$SERVER" == *":"* ]]; then
```

Comment:
`API_URL` will contain brackets if and only if it contains an IPv6 address, so you could just do

    if [[ "$API_URL" == *"["* ]]; then